src/main/java/net/glowstone/entity/objects/GlowItemFrame.java

```diff
@@ -196,10 +200,10 @@ public void setFacingDirection(BlockFace blockface) {
 
     @Override
     public void setItem(ItemStack is) {
-        if (is == null) {
-            is = new ItemStack(Material.AIR, 1);
-        }
+        is = InventoryUtil.itemOrEmpty(is);
+        is = is.clone();
         is.setAmount(1);
```

Comment:
I'm not sure what you're doing here. If a plugin uses this method with say, `ItemStack(Material.STONE, 0)` it should be treated as empty instead of updating its amount to 1 (unless vanilla does it differently?). Cloning the ItemStack should not be necessary either.